pkgs/by-name/ne/netcap/ndpi_4_0.nix

```diff
@@ -0,0 +1,22 @@
+{
+  ndpi,
+  fetchFromGitHub,
+  stdenv,
+  autoreconfHook,
+  autoconf,
+  automake,
+}:
+ndpi.overrideAttrs (finalAttrs: {
+  version = "4.0";
+  src = fetchFromGitHub {
+    owner = "ntop";
+    repo = "nDPI";
+    tag = finalAttrs.version;
+    sha256 = "0snzvlracc6s7r2pgdn0jqcc7nxjxzcivsa579h90g5ibhhplv5x";
+  };
+  configureScript = "./autogen.sh";
+  nativeBuildInputs = (builtins.filter (x: x != autoreconfHook) finalAttrs.nativeBuildInputs) ++ [
+    autoconf
+    automake
+  ];
+})
```

Comment:
Having autoreconfHook unecessarily should not cause harm or did you experience an error? Should we comment that?

Reply:
Thanks to @defelo for fixing it